things_to_do.js

```diff
@@ -1,170 +1,262 @@
+    },
+    {
+        name: 'Get a plant',
+        link: ''
+    },
+    {
+        name: 'Drink some water',
+        link: ''
+    },
+    {
+        name: 'Listen to your favorite song',
+        link: ''
+    },
+    {
+        name: 'Call your mom',
+        link: ''
+    },
+    {
+        name: 'Read about a TV trope',
+        link: 'https://tvtropes.org/'
+    },
+    {
+        name: 'Learn to draw',
+        link: 'https://drawabox.com/'
+    },
+    {
+        name: 'Change your passwords',
+        link: ''
+    },
+    {
+        name: "Cook something you've never cooked before",
+        link: ''
+    },
+    {
+        name: 'Reconnect with a friend',
+        link: ''
+    },
+    {
+        name: 'Close at least 5 tabes in your browser',
+        link: ''
+    },
+    {
+        name: 'Learn about alternatives to google search',
+        link: 'https://fossbytes.com/google-alternative-best-search-engine/'
+    },
+    {
+        name: 'Watch Gourmet Pringles attempt',
+        link: 'https://www.youtube.com/watch?v=J3xLG-whfFk'
+    },
+    {
+        name: 'Learn how to make a diamonds carbonated water',
+        link: 'https://youtu.be/n0wvDwSnzcw'
+    },
+    {
+        name: 'Relish a mechanical butterfly',
+        link: 'https://youtu.be/1gu3z7w4Vc8'
+    },
+    {
+        name: 'Think about the fact that brown does not exist',
+        link: 'https://youtu.be/wh4aWZRtTwU'
+    },
+    {
+        name: 'Learn about Snake Island',
+        link: 'https://en.wikipedia.org/wiki/Ilha_da_Queimada_Grande'
+    },
+    {
+        name: 'Crush your friend in Rock Paper Scissors',
+        link: 'https://www.youtube.com/watch?v=rudzYPHuewc'
+    },
+    {
+        name: 'Learn the best skill for your resume',
+        link: 'https://www.reddit.com/r/AskReddit/comments/1sn8ki/what_is_a_skill_someone_can_learn_in_6_months/cdzjm0e?utm_source=share&utm_medium=web2x'
+    },
+    {
+        name: 'Learn a paradox',
+        link: 'https://www.reddit.com/r/AskReddit/comments/4f50wo/whats_the_best_paradox_you_know/'
+    },
+    {
+        name: 'Read a short comics',
+        link: 'https://xkcd.com/'
+    },
+    {
+        name: "Learn about a game Ubisoft won't let you play",
+        link: 'https://www.youtube.com/watch?v=RTkxzQDo0ng'
+    },
+    {
+        name: 'Watch a strange youtube series',
+        link: 'https://www.youtube.com/watch?v=F8nUe8D5-w0&list=PLnN2bBxGARv7HS7cUsAUdfgPhKBnoUIKA'
+    },
+    {
+        name: 'Get a desk plant',
+        link: 'https://www.youtube.com/watch?v=73wGYLPXlXk'
+    },
+    {
+        name: 'Backup your photos',
+        link: ''
+    },
+    {
+        name: 'Clean your desktop',
+        link: ''
+    },
+    {
+        name: 'Clean your home screen',
+        link: ''
+    },
+    {
+        name: 'Count binary on you fingers',
+        link: 'https://www.youtube.com/watch?v=apCLHmPsC68'
+    },
+    {
+        name: 'Have you ever had a dream like this?',
```

Comment:
Maybe let's get rid of this one? :D I want to keep it wholesome and this is a bit too metal.

Reply:
changed for non-metal version :D